impl/client/src/main/javascript/web/dojo/pentaho/common/RowLimitControl.js

```diff
@@ -51,7 +51,10 @@ define(["dojo/_base/declare", "dijit/form/Select", "dijit/form/ValidationTextBox
             postCreate: function () {
               this.inherited(arguments);
               on(this.rowLimitRestrictions, 'change', lang.hitch(this, '_onSelect'));
+              // to support left/right navigation when this.rowLimitRestrictions is in a toolbar
```

Comment:
I think it would be better if the make accessible toolbar utility would implement this logic itself. For example, elements which have special LEFT/RIGHT arrow behaviors could specify an attribute to opt into a TAB mode, e.g. `data-pen-toolbar-nav-use-tabs="true"`.